src/Hng.Web/Program.cs

```diff
@@ -9,11 +14,16 @@
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
 
-var connString = builder.Configuration.GetConnectionString("DefaultConnectionString");
+var connString = builder.Configuration.GetConnectionString("DefaultConnection");
 builder.Services.AddConfiguredServices(connString);
 
 var app = builder.Build();
 
+using var scope = app.Services.CreateScope();
```

Comment:
Move line 22 to 26 into an extension method.
Remove line 24. Create a generic repository instead.